packages/desktop-client/src/components/modals/ImportTransactions.js

```diff
@@ -757,7 +852,7 @@ export default function ImportTransactions({ modalProps, options }) {
         break;
       }
 
-      let { inflow, outflow, ...finalTransaction } = trans;
+      let { inflow, outflow, inout, ...finalTransaction } = trans;
```

Comment:
```suggestion
      let { inflow, outflow, inOut, ...finalTransaction } = trans;
```

first I read this as a typo of `input`. Took me a second to understand it's actually `in out` :)